scss/styles.scss

```diff
@@ -5,7 +5,9 @@
 .body {
   margin: 0 auto;
   max-width: 1000px;
-  width: 90%
+  width: 90%;
```

Comment:
Why this width?

Reply:
Main content (<body>) centered with flex.